test/e2e/play_build_test.go

```diff
@@ -100,6 +140,44 @@ LABEL marge=mom
 		Expect(inspectData[0].Config.Labels).To(HaveKeyWithValue("homer", "dad"))
 	})
 
+	It("Check that command is expanded", func() {
+		// Setup
+		yamlDir := filepath.Join(tempdir, RandomString(12))
+		err := os.Mkdir(yamlDir, 0755)
+		Expect(err).ToNot(HaveOccurred(), "mkdir "+yamlDir)
+		err = writeYaml(testYAMLForEnvExpand, filepath.Join(yamlDir, "echo.yaml"))
+		Expect(err).ToNot(HaveOccurred())
+		app1Dir := filepath.Join(yamlDir, "foobar")
+		err = os.Mkdir(app1Dir, 0755)
+		Expect(err).ToNot(HaveOccurred())
+		err = writeYaml(playBuildFile, filepath.Join(app1Dir, "Containerfile"))
+		Expect(err).ToNot(HaveOccurred())
+		// Write a file to be copied
+		err = writeYaml(copyFile, filepath.Join(app1Dir, "copyfile"))
+		Expect(err).ToNot(HaveOccurred())
+		// Switch to temp dir and restore it afterwards
+		cwd, err := os.Getwd()
+		Expect(err).ToNot(HaveOccurred())
+		Expect(os.Chdir(yamlDir)).To(Succeed())
+		defer func() { (Expect(os.Chdir(cwd)).To(BeNil())) }()
+
+		session := podmanTest.Podman([]string{"play", "kube", "echo.yaml"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		inspect := podmanTest.Podman([]string{"container", "inspect", "echo_pod-foobar"})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect).Should(Exit(0))
+		inspectData := inspect.InspectContainerToJSON()
+		Expect(inspectData).ToNot(BeEmpty())
```

Comment:
Suggestions:
- check container logs, confirm that output == "`BAR BAR \$\(FOO\) FOOBARBAZ`"
- add more args, like `$(BAZ) ${BAZ}`, *without* a corresponding `env` yaml, but *with* `Putenv()` in the test.